src/protocols/dhcpv4/encode.c

```diff
@@ -36,11 +36,7 @@
 
 static inline bool is_encodable(fr_dict_attr_t const *root, VALUE_PAIR *vp)
 {
-	if (!vp) return false;
-	if (vp->da->flags.internal) return false;
-	if (!fr_dict_parent_common(root, vp->da, true)) return false;
-
-	return true;
+	return vp && !vp->da->flags.internal && fr_dict_parent_common(root, vp->da, true);
```

Comment:
A small comment for me personally (for a few of these changes) is that I find the more verbose version easier to read.

Reply:
Yeah, same

Reply:
That appears to be the consensus, and the more verbose version makes sense if things change--less error-prone and easier to read diffs. So be it.